examples/app/log_layers.rs

```diff
@@ -22,14 +22,23 @@ impl<S: Subscriber> Layer<S> for CustomLayer {
     }
 }
 
-fn update_subscriber(subscriber: BoxedSubscriber) -> BoxedSubscriber {
+// For an example that uses App, see log_layers_advanced.
```

Comment:
IMO we should explain why we're not using `App`, rather than trying to reference the other example.